packages/common/src/utils/dbmss/resolve-java.ts

```diff
@@ -36,13 +39,15 @@ export const resolveJavaName = (dbmsVersion: string): IJavaName => {
             ext = 'tar.gz';
     }
 
-    const javaVersion = semver.satisfies(dbmsVersion, NEO4J_JAVA_11_VERSION_RANGE)
-        ? ZULU_JAVA_VERSION.JAVA_11
-        : ZULU_JAVA_VERSION.JAVA_8;
-
-    if (process.arch !== 'x64') {
-        throw new NotSupportedError('Unsupported architecture, install Java manually');
-    }
+    const majorVersion = semver.major(dbmsVersion);
+    const neo4jJavaVersionMapping = {
+        3: ZULU_JAVA_VERSION.JAVA_8,
+        4: ZULU_JAVA_VERSION.JAVA_11,
+        5: ZULU_JAVA_VERSION.JAVA_17,
+    };
+    const [, javaVersion] = Object.entries(neo4jJavaVersionMapping).find((item) => {
+        return item[0] === majorVersion.toString();
+    }) || ['', ZULU_JAVA_VERSION.JAVA_17];
```

Comment:
```suggestion
    const javaVersion = neo4jJavaVersionMapping[majorVersion.toString()] || ZULU_JAVA_VERSION.JAVA_17;
```

This makes sense, but if we're no longer using the version range, do we really need to have it defined [in the constants file](https://github.com/neo4j-devtools/relate/pull/385/files#diff-58b12d32914affce023c39fe0d1755d2edb68d91dc918f54e03082433861c55dR67-R68)?

Reply:
No, just left forgot to remove it.